services/forwarding/forwarding.rst

```diff
@@ -0,0 +1,90 @@
+.. _doc_mailforwarding:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _`Manage Email Addresses`: http://mailsac.com/addresses
+.. _`Manage Domains`: http://mailsac.com/domains
+
+Email Forwarding
+================
+
+Forwarding allows email to be sent to a different email address or
+service (Webhook, Websocket, Slack Webhook). Private addresses and custom
+domains can be configured for email forwarding. Forwarding is not available on
+disposable email addresss.
+
+Forward to Another Mailsac Address
```

Comment:
This content is very concise and clear.

Even though it's obvious, i think adding the following info would be helpful, somewhere on this page, perhaps elsewhere.

- A private address may be created upon @mailsac.com or your own custom domain.